WordPressUI/Extensions/UIViewController+Helpers.swift

```diff
@@ -9,7 +9,9 @@ extension UIViewController {
         let visibleAsTopOnStack = navigationController?.topViewController == self && view.window != nil
         let visibleAsPresented  = view.window?.rootViewController?.presentedViewController == self
 
-        return visibleAsRoot || visibleAsTopOnStack || visibleAsPresented
+        let isPresentingAView = presentedViewController != nil
```

Comment:
Nit picking here but since we are checking on the negative anyways can we do
`let isNotPresentingViewController = presentedViewController == nil`

`return isNotPresentingViewController && (visibleAsRoot || visibleAsTopOnStack || visibleAsPresented)` ?

Reply:
Why not. 🤷‍♂ I used `isNotPresentingAView` so it's shorter and the `return` is still 1 line. 😄 edd2908